PatientWebApplication/HealthClinic.DAL/Service/OperationService.cs

```diff
@@ -55,7 +58,68 @@ public Boolean isTermNotAvailable(DoctorUser doctor, TimeSpan start, TimeSpan en
             if (hasAppointmentDoctor || hasOperationDoctor ) return true;
             return false;
         }
-       
+
+        public List<Operation> GetOperationsForPatient(int id)
+        {
+            return _operationRepository.GetOperationsForPatient(id);
+        }
+
+        public List<Operation> SimpleSearchOperations(AppointmentReportSearchDto appointmentReportSearchDto)
+        {
+            List<Operation> operations = GetOperationsForPatient(appointmentReportSearchDto.PatientId);
+
+            operations = SearchForDate(operations, appointmentReportSearchDto);
+
+            operations = SearchForDoctorNameAndSurname(operations, appointmentReportSearchDto);
+
+            operations = SearchForAppointmentType(operations, appointmentReportSearchDto);
+
+            return operations;
+
+        }
+
+        private List<Operation> SearchForDoctorNameAndSurname(List<Operation> operations, AppointmentReportSearchDto appointmentSearchDto)
+        {
+            if (!appointmentSearchDto.DoctorNameAndSurname.Equals(""))
```

Comment:
It is necessary to add a method that checks if it is an empty string, and replace it in the if statement.